lib/open_mensa/update_feeds_task.rb

```diff
@@ -13,22 +13,15 @@ def do
   private
 
   def fix_next_fetch_ats
-    disabled_feeds = Feed
-      .joins(source: :canteen)
-      .where.has { source.canteen.state == 'archived' }
-      .select(:id)
+    changes = Feed.inactive
```

Comment:
I suggest a second review of this crucial business logic part. I spotted no obvious errors but you never know.